pkg/controller/operators/catalog/operator.go

```diff
@@ -1033,6 +1042,84 @@ func (o *Operator) createInstallPlan(namespace string, subs []*v1alpha1.Subscrip
 	return reference.GetReference(res)
 }
 
+func (o *Operator) checkBundleLookups(plan *v1alpha1.InstallPlan) (bool, error) {
+	for _, bundleLookup := range plan.Status.BundleLookups {
+		if bundleLookup.BundleJob == nil {
+			configmap, job, err := configmap.LaunchBundleImage(o.opClient.KubernetesInterface(), bundleLookup.Image, "quay.io/jpeeler/bundle-init-image:latest", o.namespace)
```

Comment:
Do you mean it should remain its own image entirely? A while back I added the opm binary to the openshift image: https://github.com/operator-framework/operator-registry/commit/9d0036ae2b81ae8e531212dca88cb9feeaf128c2. For upstream I was considering adding it to the configmap-operator-registry image. 

But definitely this image needs to be changed to something official and maintained.

Reply:
Added here: https://github.com/operator-framework/operator-registry/pull/142

Reply:
I primarily meant that, code-wise, it would be nice for the image to be an optional configuration.

The other part of things is that the default should at least be under the `operator-framework` Quay org, which it looks like you've addressed already.